src/ui/public/management/section.js

```diff
@@ -20,10 +23,18 @@ export default class ManagementSection {
       index: ['id'],
       order: ['order']
     });
+    this.visible = true;
+    this.disabled = false;
+    this.tooltip = '';
+    this.url = '';
 
     assign(this, options);
   }
 
+  get visibleItems() {
+    return this.items.inOrder.filter(item => item.visible);
```

Comment:
It looks like if I specify `group` values, the `byVisible` and `byEnabled` methods that are generated return simple `Array` objects instead of references back to the `IndexedArray`. This stops us from using the `inOrder` method to maintain the sort order of the items which is important.